### PR TITLE
Learning WG: Update meeting notes 2019-10-01

### PR DESCRIPTION
**Pull request by amanjeev**
Meeting minutes

### CODE REVIEW

File: content/working-groups/learning/minutes/2019.10.01-meeting.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Learning WG 2019-10-01 sync meeting
+type: docs
+---
+# Learning WG 2019-10-01 sync meeting
+
+- Team discussed that watching same video at the same time works well so far
```
Comment (spastorino) [inline content/working-groups/learning/minutes/2019.10.01-meeting.md]: Sorry, my english is not great but ... wouldn't be better to say something like ... `Team discussed that having all of us watching the same video at the same time works well so for` like putting more emphasis that we indeed watched all the same video and discussed it.

File: content/working-groups/learning/minutes/2019.10.01-meeting.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - Everyone is [taking notes in this paper doc](https://paper.dropbox.com/doc/Ty-lecture-summary--AlwygwSe9rh1NVn2BB6CaBB9Ag-4jFj9bVOLlW7uhIOWHITX).
+  - This lecture https://github.com/rust-lang/rustc-guide/issues/375
```
Comment (spastorino) [inline content/working-groups/learning/minutes/2019.10.01-meeting.md]:
```suggestion
  - About this lecture https://github.com/rust-lang/rustc-guide/issues/375
```

File: content/working-groups/learning/minutes/2019.10.01-meeting.md (hunk 1 above)
Comment (tshepang) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: "at the same time" also implies that we watched it together

Comment (spastorino) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: But we didn't watched together at the same time. Still I just raised a doubt on the way to explain it and my english is ungreat so I'd leave it up to you. But I'm fine leaving it as is.

File: content/working-groups/learning/minutes/2019.10.01-meeting.md (hunk 2 above)
Comment (spastorino) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: My suggestion also doesn't look great. When I read "this lecture" I thought what about this lecture? seems like an incomplete phrase.

File: content/working-groups/learning/minutes/2019.10.01-meeting.md (hunk 1 above)
Comment (amanjeev) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: You are correct about this wording. I felt there was something off. Let me edit this and re-push.

Comment (amanjeev) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: Your English is good, @spastorino (it is my second language too)! Its just that my job was sloppy. :stuck_out_tongue:

Comment (amanjeev) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: How about this 

```Team discussed that watching same video by all members, one video at a time, works well for everyone so far``` ?

Comment (spastorino) [reply content/working-groups/learning/minutes/2019.10.01-meeting.md]: Seems good to me.

**Pull request merged**